Update DRED document with links.

ChuckM-Intel opened the pull request.

ChuckM-Intel committed:
Update DRED document with links.

PRMerger14 commented:
@ChuckM-Intel : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Update use-dred.md

Adding link text so that the reader knows what the destination is before following the link (in case they recognize they've been there before). Also making it clear how far they'll be taken (to a repo; blog post; whatnot).

PRMerger19 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.
